object/service.proto

```diff
@@ -688,3 +695,37 @@ message GetRangeHashResponse {
   // transmission.
   neo.fs.v2.session.ResponseVerificationHeader verify_header = 3;
 }
+
+// Replicate RPC request
+message ReplicateRequest {
+  // Object to be replicated.
+  neo.fs.v2.object.Object object = 1;
+
+  // Signature of all other request fields serialized in Protocol Buffers v3
+  // format in ascending order of fields.
+  neo.fs.v2.refs.Signature signature = 2;
+}
+
+// Replicate RPC response
+message ReplicateResponse {
+  // Operation status codes.
+  enum Code {
```

Comment:
No `ResponseMetaHeader`? No `ResponseVerificationHeader`? No common codes? I'd expect it at least to follow common status data and reuse existing codes (`ResponseMetaHeader`). Separate codes can also be very inconvenient.

Reply:
predictable questions. For now, I only doubt the need to add a signature similar to the one in the request. The rest is overhead for this simple operation

Reply:
Unified error codes are very valuable to me.

Reply:
will reuse, not yet supported cases can fallback to `INTERNAL` (like #269)

Reply:
check pls

Reply:
We can extend the set if needed.

Reply:
there is no urgent need now